.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ env:
   # This value should be greater than the time taken for the longest image pull.
   TESTCONTAINERS_PULL_PAUSE_TIMEOUT: 600
   TEST_REPORT_RETENTION_DAYS: 5
+  SEGMENT_DOWNLOAD_TIMEOUT_MINS: 5
```

Comment:
add a comment/link - it's not obvious who's the user of this since this env var is not referred to anywhere in the workflow/code.

Reply:
Done!